.gitignore

```diff
@@ -50,6 +50,8 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
+pytest_output/
+pytest_output_debug/
```

Comment:
Once PR #87 is merged we shouldn't need these lines.

Reply:
I've removed them.  Thanks.